pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -26,10 +30,16 @@ import (
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
 )
 
+const (
+	managedByLabelKey   = "networking.openshift.io/managedBy"
+	managedByLabelValue = "oc-connectivity-check-controller"
```

Comment:
I think for OpenShift we can adopt a naming convention for the field managers, so it's easier to make sense from the names.
I would recommend that the prefix should be the controller name, the name that appears in log/event so its easier to identify.